postoffice.py

```diff
@@ -12,34 +12,37 @@
 
 PASSPHRASE = sys.stdin.readline()
 
+
```

Comment:
Accidental newline?

Reply:
All of these were done by the auto code reformat in PyCharm.